scripts/create_source_tarball.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env sh
+#
+
+set -e
+
+REPO_ROOT="$(dirname "$0")"/..
+(
+    cd "$REPO_ROOT"
+    version=`grep -oP "PROJECT_VERSION \"?\K[0-9.]+(?=\")"? CMakeLists.txt`
+    commithash=`git rev-parse --short=8 HEAD`
+    committimestamp=`git show --format=%ci HEAD | head -n 1`
+    commitdate=`git show --format=%ci HEAD | head -n 1 | cut - -b1-10 | sed -e 's/-0?/./' | sed -e 's/-0?/./'`
+
+    # file exists and has zero size -> not a prerelease
+    if [ -e prerelease.txt -a ! -s prerelease.txt ]
```

Comment:
The release branch has that file and it is empty. If the file does not exist or if it contains a string, it is a prerelease (in the former case, the "prerelease portion" of the version is composed from the commit hash, in the latter, the contents of that file are used directly).